lib/rules/new-line-before-expect.js

```diff
@@ -8,14 +8,21 @@
 var hasPaddingBetweenTokens = require('../helpers/hasPaddingBetweenTokens')
 
 var blockRegexp = /^((f|x)?(it|describe))$/
+var lastExpectNode
```

Comment:
Instead of declaring  it here and resetting inside CallExpression, can we declare it in the scope it will be used. So inside CallExpression?